settings.gradle

```diff
@@ -1,3 +1,3 @@
 // sdk modules
 include ':sdk'
-include ':sdk:core', ':sdk:push'
```

Comment:
So I actually did away with the push module for now. Initially I thought having a push module separate was the way to go but getting deeper into implementation it was just too messy to try and decouple push logic entirely from the core SDK logic since they are coupled pretty fighting in the Klaviyo APIs thanks to push tokens being passed through in analytics calls